ext/DimensionalDataMakie.jl

```diff
@@ -561,7 +600,18 @@ function _lookup_to_vector(l)
     end
 end
 
+function _lookup_to_vertex_vector(l)
+    if isintervals(l)
+        bs = intervalbounds(l)
+        return @. first(bs) + (last(bs) - first(bs)) / 2
```

Comment:
Is it?  If so that would be great.   Does shiftlocus also handle converting point sampling to interval sampling?

Reply:
No you have to do that separatly: `shiftlocus(Start(), set(lookup, Intervals()))`. Points will always turn into `Intervals(Center())` by default.

Reply:
Its good to always use the methods like `shiftlocus` for things like this so we can centralise handling of the edge cases.

Reply:
ok people. I just tested this one in combination with https://github.com/rafaqz/Rasters.jl/pull/723, and things seems to work really well. So, I would like to merge this one, so that we can also fix `Rasters#main`.